sdk/data/aztables/internal/constants.go

```diff
@@ -10,7 +10,7 @@ package internal
 
 const (
 	moduleName    = "internal"
-	moduleVersion = "v0.6.1"
+	moduleVersion = "v0.5.1"
```

Comment:
Version was decremented (unclear if this is actually used anywhere).

Reply:
Fixed